.github/workflows/realm-dart-linux.yml

```diff
@@ -2,19 +2,22 @@ name: Realm Dart for Linux
 
 on:
   push:
-    branches: 
+    branches:
       - master
```

Comment:
these needs to run on any branch not just master

Reply:
It does on PRs

Reply:
yes and it should run on any other branch as well. Merging into `dev` branch should build if we have it. Now this change restricts us working only in master

Reply:
The only thing I'm changing in that line is that I remove a space at the end. I don't think you want us to do full CI on push to any branch, including feature branches we are still working on? Or maybe I don't understand your comment..

Reply:
Pardon me. I didn't read this right.